src/Cache/TwigTemplateRecorderInjector.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CacheBundle\Cache;
+
+use Sonata\Cache\Invalidation\Recorder;
+use Twig_Environment;
+use Twig_Extension;
+use Twig_Extension_InitRuntimeInterface;
```

Comment:
Please use the namespaced version (`Twig\Environment`, etc.)

Reply:
What is the lowest version of twig that we support? The namespaced aliases were [added in 1.34.0](https://github.com/twigphp/Twig/commit/5420a5e3f8a7a36022e8a7cccdde7f99ec0e14f3#diff-5e190f538c5c328c7473a5cfd3b7cd95)

Reply:
Whatever it is, we can bump it